vortex-ffi/src/duckdb/mod.rs

```diff
@@ -30,11 +29,46 @@ pub unsafe extern "C-unwind" fn vx_dtype_to_duckdb_logical_type(
 ) -> duckdb_logical_type {
     let dtype = unsafe { dtype.as_ref().vortex_expect("null dtype") };
 
-    try_or(error, ptr::null_mut(), || {
+    try_or(error, null_mut(), || {
```

Comment:
Nit, but I prefer the namespaced version just so it's clearer what the null_mut actually means